__tests__/Auth0Client/getTokenSilently.test.ts

```diff
@@ -617,30 +617,6 @@ describe('Auth0Client', () => {
           userAgent:
             'Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; AS; rv:11.0) like Gecko',
           supported: false
-        },
-        {
-          name: 'Safari 10',
-          userAgent:
-            'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/603.3.8 (KHTML, like Gecko) Version/10.1.2 Safari/603.3.8',
-          supported: false
-        },
-        {
-          name: 'Safari 11',
-          userAgent:
-            'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/604.1.28 (KHTML, like Gecko) Version/11.0 Safari/604.1.28',
-          supported: false
-        },
-        {
-          name: 'Safari 12',
-          userAgent:
-            'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0.1 Safari/605.1.15',
-          supported: false
-        },
-        {
-          name: 'Safari 12.1',
-          userAgent:
-            'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.2 Safari/605.1.15',
-          supported: true
```

Comment:
I didn't completely refactor the test here despite there being only 1 browser to test, but we might add to this later so made sense to me to leave it. What do you think? @frederikprijck

Reply:
Yes I agree!

Reply:
Codecov says otherwise 😂

Reply:
Ye we need to add one with `supported: true` .. 😬 Or rework it ?

Reply:
I can leave the Safari 12.1 test in, would that help?

Reply:
I think that could help. Wouldn't make much sense to some degree tho. Because why test only safari 12.1, and not other versions or firefox or ...  ?

Reply:
I've added a known supported browser in this test, Chrome 76, just to give us that positive and negative angle.